flexmeasures/data/migrations/versions/4b5aa7856932_added_primary_and_secondary_color_field_.py

```diff
@@ -0,0 +1,458 @@
+"""Added primary and secondary color field to account model
+
+Revision ID: 4b5aa7856932
+Revises: 126d65cbe6b4
+Create Date: 2024-08-08 13:38:17.197805
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "4b5aa7856932"
+down_revision = "126d65cbe6b4"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table("asset")
+    op.drop_table("asset_type")
+    op.drop_table("weather_sensor")
+    op.drop_table("market")
+    op.drop_table("weather_sensor_type")
+    op.drop_table("market_type")
+    op.drop_table("power")
+    op.drop_table("weather")
+    op.drop_table("price")
+    with op.batch_alter_table("account", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column("primary_color", sa.String(length=7), nullable=True)
+        )
+        batch_op.add_column(
+            sa.Column("secondary_color", sa.String(length=7), nullable=True)
+        )
+
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("account", schema=None) as batch_op:
+        batch_op.drop_column("secondary_color")
+        batch_op.drop_column("primary_color")
+
+    op.create_table(
```

Comment:
Likewise, all the downgrade instructions from this line onwards should not be part of this PR.

Reply:
Thanks for the pointer, i clear them up